Source/ui_unix/mainwindow.cpp

```diff
@@ -0,0 +1,228 @@
+#include "mainwindow.h"
+#include "settingsdialog.h"
+
+#include "openglwindow.h"
+
+#include <QFileDialog>
+#include <QTimer>
+#include <QWindow>
+
+#include "global.h"
+#include "GSH_OpenGLQt.h"
+#include "tools/PsfPlayer/Source/SH_OpenAL.h"
+
+#include "ui_mainwindow.h"
+
+#define PREFERENCE_UI_SHOWFPS "ui.showfps"
+#define PREFERENCE_UI_SHOWVIRTUALPAD "ui.showvirtualpad"
+#define PREFERENCE_AUDIO_ENABLEOUTPUT "audio.enableoutput"
+#define PREF_CGSH_OPENGL_ENABLEHIGHRESMODE        "renderer.opengl.enablehighresmode"
```

Comment:
OpenGL GS handler related preference strings are not used anywhere in this file. They can be removed.